scripts/publish-staged

```diff
@@ -113,10 +134,24 @@ while getopts ":d:p:s:b:i:a:h" opt; do
 	esac
 done
 
+if [ -n "${TARGET_OS_OPT}" ]; then
+	TARGET_OS_LIST=()
+	for os in `echo "${TARGET_OS_OPT}" | tr ',' ' '`; do
```

Comment:
Can you change the backticks to `$()`?

Or, alternately: Wouldn't it be easier to do this
```bash
oldIFS="${IFS}"
IFS=","
for os in "${TARGET_OS_OPT}"; do
#...
done
IFS="${oldIFS}"
```
?